### DISCUSSION TITLE
Support for MUI components props directly

### DISCUSSION THREAD

**Discussion by venkatesh1146** [manojadams/mui-forms#55] [Ideas]
Hi @manojadams 

As we are using the MUI components for the forms, why don't we support the props from schema to the MUI component.

Ex:
If I want a multi-select field in the Form and we are using the [MUI select  ](https://mui.com/material-ui/react-select/) we can use props to make it [multi-select](https://mui.com/material-ui/react-select/#multiple-select)

This also gives the flexibility to customize the MUI UI components.

**Comment by manojadams** [comment]
## Description
There is a plan to enable props through json schema but multi-select will be taken as a separate control if planned as it can have dynamic behaviour and can be used in many different ways.

## Post Action: 
Enable custom props through json schema fields.

**Comment by manojadams** [comment]
@venkatesh1146 @venkateshpullaganti This is now supported through custom props.
Ref: https://mui-forms.vercel.app/custom-props

**Reply by manojadams** [reply]
Version: 2.4.9

**Reply by venkatesh1146** [reply]
Got it @manojadams 
Thank you 🙌
